app/src/main/java/com/devaseemsharma/taskyapp/authentication/presentation/login/LoginViewModel.kt

```diff
@@ -0,0 +1,76 @@
+package com.devaseemsharma.taskyapp.authentication.presentation.login
+
+import androidx.compose.runtime.State
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import com.devaseemsharma.taskyapp.authentication.presentation.TextFieldState
+import dagger.hilt.android.lifecycle.HiltViewModel
+import org.w3c.dom.Text
+import javax.inject.Inject
+
+@HiltViewModel
+class LoginViewModel @Inject constructor(
+) : ViewModel() {
+    private val _emailAddress = mutableStateOf(TextFieldState(
+        hint = "Email address"
+    ))
+    val emailAddress: State<TextFieldState> = _emailAddress
+
+    private val _passwordText = mutableStateOf(TextFieldState(
+        hint = "Password"
+    ))
+    val passwordText: State<TextFieldState> = _passwordText
```

Comment:
If you prefer to use Compose states in your ViewModel, this is shorter and achieves the same:
```
var emailAddress by mutableStateOf(TextFieldState(hint = "Email address"))
    private set
```